src/devices/BoardLed/BoardLed.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Iot.Device.BoardLed
+{
+    /// <summary>
+    /// On-board LED on the device.
+    /// </summary>
+    public class BoardLed
+    {
+        private const string DefaultDevicePath = "/sys/class/leds";
+
+        /// <summary>
+        /// The name of the LED.
+        /// </summary>
+        public string Name { get; }
+
+        /// <summary>
+        /// The kernel controls the trigger of the LED.
```

Comment:
would be useful to add some remarks on what does this mean and how to use it

Reply:
also there is an API to enumerate triggers and get 1 trigger? Presumably this api will return some kind of concatenated version of the other one? Does it make sense to hide one or another?

Reply:
For example, the power red light of Raspberry Pi, whose trigger is `default-on`(keep lighting up), and the green is `mmc0`(Light up when when writing SD card). If I want to operate the LED, I need to remove the trigger, which is to set its trigger to `none`. In addition, the kernel provides some default triggers that can be changed by setting their triggers.

Reply:
@krwq btw: Please review #664 if you have time. Only by merging can I add `SoundDevice` next.😄

Reply:
Please add example (if not present already) showing how this should be used since the API is not exactly obvious